Parse/Infrastructure/Data/ParseDataDecoder.cs

```diff
@@ -20,44 +20,44 @@ public class ParseDataDecoder : IParseDataDecoder
 
     public object Decode(object data, IServiceHub serviceHub)
     {
-            return data switch
+        return data switch
+        {
+            null => default,
+            IDictionary<string, object> { } dictionary when dictionary.ContainsKey("__op") => ParseFieldOperations.Decode(dictionary),
+
+            IDictionary<string, object> { } dictionary when dictionary.TryGetValue("__type", out var type) && Types.Contains(type) => type switch
             {
```

Comment:
Could you please revert all code style changes in code that is unrelated to this issue. It seems that there hasn't been any code change other than the indentation, haven't looked closely though.

Can we solve this with a linter? Is there a linter already in place for this repo?

Reply:
I, do not know what *linters* would be but I believe [this](https://github.com/parse-community/Parse-SDK-dotNET/pull/393#issuecomment-2535314741) would be related.

For the code itself, would you mean I would; remove these and submit in a different PR?

Reply:
Not sure I follow. I was referring to changes that are only related to spacing for example. If they are not in code that is related to the issue the PR addresses, they should be revered. An example is the file I tagged above, there seem to be no code changes, only indentation or line-break changes. These changes should be reverted in this PR. They don't need to be added in a separate PR, a linter should take care of that.

Reply:
Oh. I'm afraid my knowledge is quite sparse when it comes to "lints" then..

Reply:
See https://github.com/parse-community/Parse-SDK-dotNET/pull/404#pullrequestreview-2523532572